lib/Dialect/Poly/IR/PolyOps.cpp

```diff
@@ -32,7 +32,7 @@ LogicalResult FromTensorOp::verify() {
   }
 
   APInt coefficientModulus = ring.coefficientModulus();
-  unsigned cmodBitWidth = coefficientModulus.logBase2();
+  unsigned cmodBitWidth = coefficientModulus.ceilLogBase2();
```

Comment:
fyi: new bug fix. after incorporating the verifier PR, and with the new modulus I was testing for the change, I hit a bug that a size `32` integer coeff array was not valid, `logBase2` returns floor of the log. @j2kun